app/javascript/components/PlaylistRamp.jsx

```diff
@@ -94,6 +105,34 @@ const Ramp = ({
     }
   };
 
+  const expandBtn = {
+    paddingLeft: '2px',
+    cursor: 'pointer'
+  };
+
+  const wordCount = 32;
+  const words = comment ? comment.split(' ') : [];
+
+  function prepInitialDescription() {
+    if (words !== undefined && words.length > 0) {
+      clearInterval(descriptionCheck);
+      let desc = words.length > wordCount
+        ? `${words.slice(0, wordCount).join(' ')}...`
+        : words.join(' ');
+
+      setDescription(desc);
+    } else if (words.length === 0) {
+      clearInterval(descriptionCheck);
+    }
+  }
+
+  const handleClick = () => {
```

Comment:
How about making this function name more specific?
```suggestion
  const handleDescriptionMoreLessClick = () => {
```